[2.12.0] Update `ui-sql-cache` & Server-Side Pagination Content

sunilarjun opened the pull request:
Updating pages regarding `ui-sql-cache` feature flag combining functionality with Server-Side Pagination, and becoming enabled by default in Rancher v2.12.0. Also did a general update to the FF page as incorrect attributes were being used to display FF settings: `true` --> `Active`, `false` --> `Disabled`.

sunilarjun commented:
@richard-cox Hello! Referenced these tickets [[#48691](https://github.com/rancher/rancher/issues/48691), [#12975](https://github.com/rancher/dashboard/issues/12975)] to update content regarding the `ui-sql-cache` and SSP FF update, please let me know if there are any more content sections to add or information needed in these pages. Thank you!

richard-cox commented on versioned_docs/version-2.12/how-to-guides/advanced-user-guides/enable-experimental-features/ui-server-side-pagination.md:
```diff
@@ -6,27 +6,25 @@ title: UI Server-Side Pagination
-
-
-UI server-side pagination caching provides an optional SQLite-backed cache of Kubernetes objects to improve performance. This unlocks sorting, filtering and pagination features used by the UI to restrict the amount of resources it fetches and stores in browser memory. These features are primarily used to improve list performance for resources with high counts.
+The feature flag `ui-sql-cache` is enabled by default and implements the UI server-side pagination caching. The functionality provides an SQLite-backed cache of Kubernetes objects to improve performance. This adds sorting, filtering and pagination features used by the UI to restrict the amount of resources it fetches and stores in browser memory. These features are primarily used to improve list  ...
```
```suggestion
The feature flag `ui-sql-cache` is enabled by default and provides an SQLite-backed cache of Kubernetes objects. This unlocks sorting, filtering and pagination features used by the UI to restrict the amount of resources it fetches and stores in browser memory, and also monitors. These features are primarily used to improve list performance for resources with high counts.
```

richard-cox commented on versioned_docs/version-2.12/how-to-guides/advanced-user-guides/enable-experimental-features/ui-server-side-pagination.md:
```diff
@@ -39,3 +37,31 @@ Secrets and security Tokens are always encrypted regardless of the above setting
 This initial release improves the performance of Pods, Secrets, Nodes and ConfigMaps in the Cluster Explorer pages, and most resources in the Explorer's **More Resources** section.
```
```suggestion
This initial release improves the performance of most pages used to view, create or edit resources within the `local` or downstream clusters i.e. the Cluster Explorer view.

Areas outside of the Cluster Explorer are not yet covered by this feature.
```

richard-cox commented on versioned_docs/version-2.12/how-to-guides/advanced-user-guides/enable-experimental-features/ui-server-side-pagination.md:
```diff
@@ -39,3 +37,31 @@ Secrets and security Tokens are always encrypted regardless of the above setting
 This initial release improves the performance of Pods, Secrets, Nodes and ConfigMaps in the Cluster Explorer pages, and most resources in the Explorer's **More Resources** section.
 
 Pages can't be automatically refreshed. You can manually refresh table contents by clicking the **Refresh** button.
```
```suggestion
Resources in Lists are automatically updated, however not instantaneously. 
```

richard-cox commented on versioned_docs/version-2.12/how-to-guides/advanced-user-guides/enable-experimental-features/ui-server-side-pagination.md:
```diff
@@ -39,3 +37,31 @@ Secrets and security Tokens are always encrypted regardless of the above setting
+
+### Regressions
+
+There were performance regressions following the resource cache support changes, and they are outlined below:
```
```suggestion
The follow regressions occur when the feature is enabled. These mainly revolve around different sort or filter behaviours in affected lists
```

richard-cox commented on docs/how-to-guides/advanced-user-guides/enable-experimental-features/ui-server-side-pagination.md:
```diff
@@ -6,27 +6,25 @@ title: UI Server-Side Pagination
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/enable-experimental-features/ui-server-side-pagination"/>
 </head>
 
-:::caution
```
i've made the comments the wrong way around, but those in `versioned_docs/version-2.12/how-to-guides/advanced-user-guides/enable-experimental-features/ui-server-side-pagination.md` need to be applied here

richard-cox reviewed (changes requested):
A few minor tweaks required.

@moio is this the location where the change in disk requirements needs to be mentioned?

sunilarjun commented:
> A few minor tweaks required.
> 
> @moio is this the location where the change in disk requirements needs to be mentioned?

Hello thank you for reviewing! I've updated with respect to the feedback and also moved the pages out of the experimental section, please let me know if any more edits are needed.

moio commented on docs/how-to-guides/advanced-user-guides/ui-server-side-pagination.md:
```diff
@@ -0,0 +1,77 @@
+
+SQLite-backed caching persists copies of any cached Kubernetes objects to disk. See [Encrypting SQLite-backed Caching](#encrypting-sqlite-backed-caches) if this is a security concern.
+
+## Enabling via UI: `ui-sql-cache` and Server-Side Pagination
```
```suggestion
## Enabling Server-Side Pagination
```

Unless we are also describing how to do so without the UI, it does not make sense to overspecify it in the title. Moreover the `ui-sql-cache` flag is an implementation detail repeated below, so I would not have it in the title.

Similarly for the "Disabling" part

richard-cox replied:
I've validated that this change made it in, resolving (Silvio please re-open if you disagree)

richard-cox commented on docs/how-to-guides/advanced-user-guides/ui-server-side-pagination.md:
```diff
@@ -0,0 +1,77 @@
+
+## Known Limitations of UI Server-Side Pagination
+
+- This initial release improves the performance of most pages used to view, create or edit resources within the `local` or downstream clusters i.e. the Cluster Explorer view.
```
Change looks good to me, all good your side @moio?

richard-cox replied:
another derp, sorry, hadn't seen the approval. marking this as resolved
